_data/other.yml

```diff
@@ -147,6 +147,15 @@ websites:
       sms: Yes
       doc: https://secure.ssa.gov/RIL/HpsView.do#question3
 
+    - name: Webmin
+      url: http://www.webmin.com/
+      img: webmin.png
+      tfa: Yes
+      software: Yes
+      sms: Yes
+      phone: Yes
```

Comment:
When you say `phone` here, it implies there's a phone call with voice or keypad prompt...I don't think that's accurate here.